docs/src/web-client/creating_multiple_notes_tutorial.md

```diff
@@ -0,0 +1,315 @@
+const faucet = await client.newFaucet(
+  AccountStorageMode.public(),
+  false,
+  "MID",
+  8,
+  BigInt(1_000_000),
+);
+console.log("Alice:", alice.id().toString());
+console.log("Faucet:", faucet.id().toString());
+
+// ── mint 10 000 MID to Alice ──────────────────────────────────────────────────────
+await client.submitTransaction(
+  await client.newTransaction(
+    faucet.id(),
+    client.newMintTransactionRequest(
+      alice.id(),
+      faucet.id(),
+      NoteType.Public,
+      BigInt(10_000),
+    ),
+  ),
+  prover,
+);
+await new Promise((r) => setTimeout(r, 10_000));
+await client.syncState();
+
+// ── consume the freshly minted notes ──────────────────────────────────────────────
+const noteIds = (await client.getConsumableNotes(alice.id())).map((rec) =>
+  rec.inputNoteRecord().id().toString(),
+);
+
+await client.submitTransaction(
+  await client.newTransaction(
+    alice.id(),
+    client.newConsumeTransactionRequest(noteIds),
+  ),
+  prover,
+);
+await client.syncState();
+```
+
+## Step 5 — Build and Create P2ID notes
+
+Add the following code to the `multiSendWithDelegatedProver` function. This code defines three recipient addresses, builds three P2ID notes with 100 `MID` each, and then creates all three notes in the same transaction.
+
+```ts
+// ── build 3 P2ID notes (100 MID each) ─────────────────────────────────────────────
+const recipientAddresses = [
+  "0x3477d1532b97101000006f79009dda",
+  "0x13eb7f06cc675f20000d28ad4256e9",
+  "0x3a2ff6a4b1628120000d8a3650894b",
+];
+
+const script = client.compileNoteScript(P2ID_NOTE_SCRIPT);
+
+const assets = new NoteAssets([new FungibleAsset(faucet.id(), BigInt(100))]);
+const metadata = new NoteMetadata(
+  alice.id(),
+  NoteType.Public,
+  NoteTag.fromAccountId(alice.id(), NoteExecutionMode.newLocal()),
+  NoteExecutionHint.always(),
+);
+
+const p2idNotes = recipientAddresses.map((addr) => {
+  // The note serial number should be a randomly generated number
+  let serialNumber = Word.newFromFelts([
+    new Felt(BigInt(Math.floor(Math.random() * 0x1_0000_0000))),
+    new Felt(BigInt(Math.floor(Math.random() * 0x1_0000_0000))),
+    new Felt(BigInt(Math.floor(Math.random() * 0x1_0000_0000))),
+    new Felt(BigInt(Math.floor(Math.random() * 0x1_0000_0000))),
+  ]);
+
+  const acct = AccountId.fromHex(addr);
+  const inputs = new NoteInputs(new FeltArray([acct.suffix(), acct.prefix()]));
+
+  let note = new Note(
+    assets,
+    metadata,
+    new NoteRecipient(serialNumber, script, inputs),
+  );
+
+  return OutputNote.full(note);
+});
+
+let transaction = await client.newTransaction(
+  alice.id(),
+  new TransactionRequestBuilder()
+    .withOwnOutputNotes(new OutputNotesArray(p2idNotes))
+    .build(),
+);
+
+// ── create all P2ID notes ───────────────────────────────────────────────────────────────
+await client.submitTransaction(transaction, prover);
+
+console.log("All notes created ✅");
```

Comment:
Can you say, `All notes created and on-chain ready for consumption` or something similar? 

And can you show how long it took?

<img width="530" alt="Screenshot 2025-05-23 at 18 13 17" src="https://github.com/user-attachments/assets/70c19ff6-64f4-4215-a096-e5c40d39419e" />

And in general, can you make the logs a bit more descriptive. 

`Client created and syncs at block ...`
`Alice's account created, ID: ...`
`Alice's account funded with ...`